docs/contrib/issues.md

```diff
@@ -0,0 +1,19 @@
+# Reporting issues
+
+If you notice an issue (a typo, a grammar glitch, or missing or incorrect information), here's how you can let us know:
+
+Go to [our issue tracker]({{config.repo_url}}/issues) and click *New issue*.
+
+![GitHub issues list with the "New issue" button shown in green](assets/issue-list.png)
+
+Select the issue type that most closely matches the type of issue that you are reporting.
+
+![GitHub "New issue" selection showing different types of issues](assets/issue-type-select.png)
+
+Be sure to fill *all* the required fields (the ones marked with a red asterisk), and fill the optional ones as necessary.
```

Comment:
Personal preference: Drop the `the` from `the required fields`, and replace `fill` with `fill in`.

Reply:
```suggestion
Be sure to fill *all* required fields (the ones marked with a red asterisk), and fill in the optional ones as necessary.
```